faq.rst

```diff
@@ -287,29 +293,52 @@ Then set it as the upstream::
 How do I compare my local repository to a remote repository?
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-To display the list of changes that are in your local repository, but not
-in the remote, use::
+You have in your clone a copy of the state of the branches in the remote
+repository.  This state is updated any time you do a ``git pull`` or
+``git fetch``.  You can find out if your local copy is up to date with
+its origin by running::
+
+    $ git remote show origin
```

Comment:
If you do so, please add an information how to sync one's fork with current master as well and how to rebase patches, imho. Both of these are valuable information.